tf2onnx/onnx_opset/nn.py

```diff
@@ -351,7 +351,12 @@ def version_1(cls, ctx, node, **kwargs):
         # prefix with batch dim of [1] to satisfy rank requirements
         input_shape = ctx.get_shape(node.input[0])
         if len(input_shape) == spatial + 1:
-            ctx.insert_new_node_on_input(node, "Unsqueeze", node.input[0], axes=[0])
```

Comment:
Can you remove this if/else and use graphbuilder in all cases?  Seems like it should be fine.